packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -34,6 +34,9 @@ abstract class WebViewPlatformCallbacksHandler {
   /// /// Only works when [WebSettings.hasProgressTracking] is set to `true`.
   void onProgress(int progress);
 
+  ///Invoked by [WebViewPlatformController] when a user is swiping to reload a webview. //@pullToRefresh
+  Future<void> onPullToRefresh(); //@pullToRefresh
```

Comment:
This is a breaking change, so adding this will need to wait for https://github.com/flutter/flutter/issues/94051 to be fixed.